pkg/inventory/model/journal.go

```diff
@@ -29,11 +39,40 @@ type Event struct {
 	Updated Model
 }
 
+//
+// String representation.
+func (r *Event) String() string {
+	action := "unknown"
+	switch r.Action {
+	case Parity:
+		action = "parity"
+	case Error:
+		action = "error"
+	case End:
+		action = "end"
+	case Created:
+		action = "created"
+	case Updated:
+		action = "updated"
+	case Deleted:
+		action = "deleted"
+	}
+	model := ""
+	if r.Model != nil {
+		model = r.Model.String()
+	}
+	return fmt.Sprintf(
+		"event-%.4d: %s model=%s",
+		r.ID,
+		action,
+		model)
+}
+
 //
 // Event handler.
 type EventHandler interface {
 	// Watch has started.
-	Started()
+	Started(watchID uint64)
```

Comment:
Send the watch ID.
Mainly used for _remote_ watch.  The server sends this event to the (remote) client which will set the same watch ID on it's end.  This way the watch on both ends has the same ID.  helpful for correlation.